src/Symfony/Component/Serializer/Normalizer/AbstractNormalizer.php

```diff
@@ -112,6 +112,12 @@ abstract class AbstractNormalizer implements NormalizerInterface, DenormalizerIn
      */
     public const IGNORED_ATTRIBUTES = 'ignored_attributes';
 
+    /**
+     * Flag to prevent fields that are nullable to receive a null
+     * fallback value when no parameters are provided for it.
+     */
+    public const PREVENT_NULLABLE_FALLBACK = 'prevent_nullable_fallback';
```

Comment:
Could you update the `AbstractNormalizerContextBuilder` as well to handle that new context option?

Reply:
@mtarld Wasn't aware of the context builder, thanks for the hint. I've added the new option to the context builder and extended the existing tests.